code/game/area/areas/ruins/lavaland.dm

```diff
@@ -99,8 +105,10 @@
 	power_light = FALSE
 //ash walker nest
 /area/ruin/unpowered/ash_walkers
+	name = "\improper Ash Walkers' Nest"
 	icon_state = "red"
 
 /area/ruin/unpowered/ratvar
+	name = "\improper Ratvar's Ruin"
```

Comment:
this is a resting place the name ratvar's ruin is again really weird

Reply:
I'll just call it "Ratvar's Rest" or "Ratvar's Final Resting Place", although the latter is a bit more over-the-top.